pkg/registry/styles/projects.less

```diff
@@ -19,6 +63,15 @@ table.projects-listing {
         cursor: pointer;
     }
 
+    tbody.inner-project-listing-add {
```

Comment:
I see no mention of 'inner-project-listing-add' outside of this less file. I think this change is unnecessary.